dgraphtest/cluster.go

```diff
@@ -406,13 +407,19 @@ func (gc *GrpcClient) DropAll() error {
 }
 
 // Mutate performs a given mutation in a txn
-func (gc *GrpcClient) Mutate(rdfs string) (*api.Response, error) {
+func (gc *GrpcClient) Mutate(rdfs string, SetNQuads bool) (*api.Response, error) {
```

Comment:
wouldn't it be better if we take setNQuads and delNquads as args? Maybe, we can take the api.Mutation as arg itself.

Reply:
I think we should take api.Mutation as arg